src/components/AppElements/components/CodeEditor.vue

```diff
@@ -249,6 +244,7 @@ export default {
 		...mapGetters({ menus: "getMenu" }),
 		...mapGetters([
 			"getHtmlGeneratorConfigObject",
+			"getImportSettings",
```

Comment:
missing methods tab for the html import (see editor for example)

Reply:
done